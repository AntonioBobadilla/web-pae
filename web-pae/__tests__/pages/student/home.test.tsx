import Home from '@/pages/student/home';
import '@testing-library/jest-dom';
import { render } from '@testing-library/react';
import React from 'react';

describe('Student Home', () => {
  it('renders Student home page unchanged', () => {
    const { container } = render(<Home />);
    expect(container).toMatchSnapshot();
  });
});
