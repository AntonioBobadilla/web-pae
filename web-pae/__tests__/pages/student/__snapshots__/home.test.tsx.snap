// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Student Home renders Student home page unchanged 1`] = `
<div>
  <div
    class="cards"
  >
    <div
      class="card"
    >
      <div
        class="box"
        href="/student/profile"
        role="button"
      >
        <img
          class="image"
          src="/images/student-login-image.jpg"
        />
        <p
          class="description"
        >
           
          Perfil
           
        </p>
      </div>
    </div>
    <div
      class="card"
    >
      <div
        class="box"
        href="/student/schedule-tutoring"
        role="button"
      >
        <img
          class="image"
          src="/images/tutor-login-image.jpg"
        />
        <p
          class="description"
        >
           
          Agendar asesor√≠a
           
        </p>
      </div>
    </div>
  </div>
</div>
`;
